src/components/common/names-capture.vue

```diff
@@ -97,6 +97,7 @@
                               @blur="handleBlur()"
                               @input="editChoices('name1', $event, true)"
                               :filled="getIsAssumedName"
+                              :class="!getIsAssumedName ? 'read-only-mode' : ''"
```

Comment:
I think this could be simplified using vue class binding since there is no alternative class used in the ternary, ie
`{ 'read-only-mode': !getIsAssumedName }` or something along those lines.

Reply:
sure I can update that